test-utils/test-utils-core/src/main/java/uk/gov/justice/services/test/utils/core/messaging/MessageProducerClient.java

```diff
@@ -86,8 +103,7 @@ public void sendMessage(final String commandName, final JsonEnvelope jsonEnvelop
             throw new RuntimeException("Message producer not started. Please call startProducer(...) first.");
         }
 
-        @SuppressWarnings("deprecation")
-        final String json = jsonEnvelope.toDebugStringPrettyPrint();
+        @SuppressWarnings("deprecation") final String json = jsonEnvelope.toDebugStringPrettyPrint();
```

Comment:
Should this be in the catch? We only use the debug string when we have an exception?

Reply:
No - the debug string is used to create the TextMessage Object that is sent